os/kernel/debug/sysdbg.c

```diff
@@ -208,20 +208,27 @@ static void simulate_undef_abort(void)
 
 static void show_usage(void)
 {
-	dbg("USAGE:\n");
-	dbg("sysdbg enable_monitor  /* Allocates memory and enables monitoring */\n");
-	dbg("sysdbg disable_monitor /* Frees the memory and disables monitoring */\n");
-	dbg("sysdbg read   /* Reads /dev/sysdbg to view debug information*/\n");
-	dbg("sysdbg max_task_count <count> /* Enter max_task_count */\n");
-	dbg("sysdbg max_irq_count <count> /* Enter max_irq_count */\n");
-	dbg("sysdbg max_sem_count <count> /* Enter max_sem_count */\n");
-	dbg("sysdbg data_abort /* Simulate data abort */\n");
-	dbg("sysdbg prefetch_abort /* Simulate Prefetch abort */\n");
-	dbg("sysdbg undef_abort  /* Simulate Undefined abort */\n");
-	dbg("sysdbg usr_assert /* Simulate User Assert */\n");
-	dbg("sysdbg dump_stack /* Dumps the current task stack */\n");
-	dbg("sysdbg dump_allstack /* Dumps the stack of all tasks */\n");
-	dbg("sysdbg help /* Display the help */\n");
+	dbg("\n");
+	dbg(" ______________________________________________________________________________\n");
```

Comment:
Why not combine these 2 lines of code as `dbg("\n______________________________________\n");`